### ISSUE TITLE
runtime errors in Sprig do not show the proper information

### ISSUE THREAD

**Issue by snoglobe** [hackclub/sprig#1734]
input:
```js
function hee() {
  throw new Error('haa')
}

function hoo() {
  hee()
}

function hi() {
  hoo()
}

hi()
```

output:
```
Error: haa
    at hee (index.ts:4:9)
    at hoo (index.ts:8:3)
    at hi (index.ts:12:3)
    at anonymous (index.ts:12:1)
```

every line in the stack trace is off by 2

**Comment by JosiasAurel** [comment]
what's the status of the issue @snoglobe 👀 ?

**Comment by DevIos01** [comment]
This has already been fixed and merged correctly at #1979, so I am closing this issue 🫡
